tests/model_serving/model_server/llmd/api_compat/auth.py

```
@@ -0,0 +1,116 @@
+        llmisvc: LLMInferenceService,
+        sa_name: str | None = None,
+    ) -> None:
+        self._client = client
+        self._llmisvc = llmisvc
+        self._sa_name = sa_name or f"{llmisvc.name}-compat-sa"
+        self._stack: ExitStack | None = None
+        self._token: str | None = None
+
+    def __enter__(self) -> Self:
+        self._stack = ExitStack()
+        sa = self._stack.enter_context(  # noqa: FCN001
```

Ruff (0.16.0)

[warning] Invalid rule code in `# noqa`: FCN001

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)